pkg/network/server.go

```diff
@@ -664,32 +664,33 @@ func (s *Server) handleMessage(peer Peer, msg *Message) error {
 }
 
 func (s *Server) handleNewPayload(p *consensus.Payload) {
-	s.relayInventory(payload.ConsensusType, p.Hash())
+	s.relayInventoryCmd(CMDInv, payload.ConsensusType, p.Hash())
 }
 
 func (s *Server) requestTx(hashes ...util.Uint256) {
 	if len(hashes) == 0 {
 		return
 	}
 
-	s.relayInventory(payload.TXType, hashes...)
+	s.relayInventoryCmd(CMDGetData, payload.TXType, hashes...)
 }
 
-func (s *Server) relayInventory(t payload.InventoryType, hashes ...util.Uint256) {
+func (s *Server) relayInventoryCmd(cmd CommandType, t payload.InventoryType, hashes ...util.Uint256) {
 	payload := payload.NewInventory(t, hashes)
 	msg := NewMessage(s.Net, CMDInv, payload)
```

Comment:
Replace `CMDInv` with `cmd` ?

Reply:
It's strange that linter hasn't catch this. I was sure it proposes to replace unused parameter with `_`.

Reply:
Ooopsie…